javatests/arcs/core/entity/DummyVariableEntity.kt

```diff
@@ -0,0 +1,52 @@
+package arcs.core.entity
+
+import arcs.core.data.*
+
+class DummyVariableEntity : VariableEntityBase(ENTITY_CLASS_NAME, SCHEMA), Storable {
+    var text: String? by SingletonProperty()
+    var ref: Reference<DummyEntity>? by SingletonProperty()
+    var bools: Set<Boolean> by CollectionProperty()
+    var nums: Set<Double> by CollectionProperty()
+
+    private val nestedEntitySpecs = mapOf(
+        DummyEntity.SCHEMA_HASH to DummyEntity
+    )
+
+    fun getSingletonValueForTest(field: String) = super.getSingletonValue(field)
+
+    fun getCollectionValueForTest(field: String) = super.getCollectionValue(field)
+
+    fun setSingletonValueForTest(field: String, value: Any?) =
```

Comment:
why do we need this?

Reply:
Yeah, a lot of these methods are not used; I'll remove.